public/index.html

```diff
@@ -11,6 +11,7 @@
     -->
 	<link rel="manifest" href="%PUBLIC_URL%/manifest.json">
 	<link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+	<link rel="stylesheet" href="%PUBLIC_URL%/fonts/fonts.css">
```

Comment:
You shouldn't load custom fonts in an external CSS file, because the browser will only load it after 2 roundtrips with the server:

```
client                 server
  |                          |
  | load index.html -------> |
  | <------------------------|
  |                          |
  | load fonts.css --------> |
  | <------------------------|
  |                          |
  | load font.otf ---------> |
  | <------------------------|
```

The consequence, for the end user, is a long Flash Of Unstyled Text (FOUT). You can reduce it by inlining fonts.css in the index.html.

More informatin about FOUT: https://www.ebayinc.com/stories/blogs/tech/ebays-font-loading-strategy/

Reply:
Done